modules/nf-core/jvarkit/vcfpolyx/main.nf

```diff
@@ -23,24 +23,28 @@ process JVARKIT_VCFPOLYX {
     task.ext.when == null || task.ext.when
 
     script:
-    def args1  = task.ext.args1 ?: ''
-    def args2  = meta.vcfpolyx_args ?: (task.ext.args2 ?: ' --tag POLYX --max-repeats 10 ')
-    def args3  = task.ext.args3 ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args1         = task.ext.args1 ?: ''
+    def args2         = task.ext.args2 ?: ''
+    def args3         = task.ext.args3 ?: ''
+    def prefix        = task.ext.prefix ?: "${meta.id}"
+    def regions_cmd   = regions_file ? (tbi ? " --regions-file" : " --targets-file") + " '${regions_file}' " : ""
```

Comment:
I have a suspicion that this won't work, because the $ won't be evaluated inside the single quotes, but I'm not sure.
```suggestion
    def regions_cmd   = regions_file ? (tbi ? " --regions-file" : " --targets-file") + ' "${regions_file}" ' : ""
```

Reply:
it won't be evaluated in BASH, and that's what we want.